service/src/main/java/org/sunbird/actor/user/validator/UserRequestValidator.java

```diff
@@ -80,6 +81,20 @@ private void validatePassword(String password) {
     }
   }
 
+  private void validateFirstName(Request userRequest) {
```

Comment:
it is not validating... it is just updating the format... 
Do rename the method to formatFirstName

Reply:
Renamed the method to formatFirstName